Content.Client/Storage/UI/StorageWindow.xaml.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using Content.Client.Items.Components;
+using Robust.Client.AutoGenerated;
+using Robust.Client.GameObjects;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Robust.Shared.Maths;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Utility;
+using Robust.Client.Utility;
+using Robust.Client.ResourceManagement;
+using Content.Client.Stylesheets;
+
+namespace Content.Client.Storage.UI
+{
+    [GenerateTypedNameReferences]
+    public sealed partial class StorageWindow : SS14Window
+    {
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+        [Dependency] private readonly IResourceCache _resCache = default!;
+
+        private Font _weightFont;
+        private Font _weightFontSmall;
```

Comment:
Put these in the Stylesheet and use the subclass to change between normal and small.

Reply:
I've added style classes for the label & weight-icon, so the font & color is now set in the stylesheet.